packages/client/components/OverflowAvatar.tsx

```diff
@@ -55,30 +57,23 @@ interface Props {
 }
 
 const OverflowAvatar = (props: Props) => {
-  const {
-    overflowCount,
-    offset,
-    status,
-    onTransitionEnd,
-    isAnimated,
-    onClick,
-    width,
-    borderColor
-  } = props
+  const {overflowCount, offset, status, onTransitionEnd, isAnimated, onClick, width, borderColor} =
+    props
   const ref = useRef<HTMLDivElement>(null)
   const label = overflowCount >= 99 ? 99 : overflowCount
   useResizeFontForElement<HTMLDivElement>(ref, label, 11, 18, 4)
   return (
-    <Wrapper offset={offset} onClick={onClick}>
+    <Wrapper offset={offset}>
       <OverflowCount
         width={width}
         ref={ref}
         status={status}
         onTransitionEnd={onTransitionEnd}
         isAnimated={isAnimated}
         borderColor={borderColor}
+        onClick={onClick}
       >
-        {label}
+        +{label}
```

Comment:
added `+` before overflow count (as DashboardAvatars did)
It seems to me more understandable from a UX point of view but I'm not sure about the current guideline.